stage1_fly/run/main.go

```diff
@@ -218,8 +218,16 @@ func stage1() int {
 		return 1
 	}
 
-	// TODO: insert environment from manifest
-	env := []string{"PATH=/bin:/sbin:/usr/bin:/usr/local/bin"}
+	defaultEnv := []string{"PATH=/bin:/sbin:/usr/bin:/usr/local/bin"}
+	envList := p.Manifest.Apps[0].App.Environment
+
+	env := make([]string, len(envList))
+	for _, e := range envList {
+		if _, exists := envList.Get(defaultEnv[0]); !exists {
+			env = append(env, defaultEnv[0])
```

Comment:
This `append` should not be in the `for` loop because this adds the "$PATH" environment variable as many times as there are variables defined in the ACI manifest.